docs/en/operation/Admin-CLI.md

```diff
@@ -26,27 +27,30 @@ Usage: alluxio fsadmin [generic options]
 
 ### backup
 
-The `backup` command creates a backup of Alluxio metadata.
+The `backup` command creates a backup of Alluxio metadata. By default, this is the '$ALLUXIO_HOME/alluxio_backups'
+on the primary Alluxio Master.
 
-Back up to the default backup folder (configured by `alluxio.master.backup.directory`)
+Back up to the default backup folder (this default can be configured by setting `alluxio.master.backup.directory`)
 ```
 ./bin/alluxio fsadmin backup
-Successfully backed up journal to hdfs://mycluster/opt/alluxio/backups/alluxio-backup-2018-5-29-1527644810.gz
+Successfully backed up journal to /opt/alluxio/alluxio_backups/alluxio-backup-2019-06-13-1560446387260.gz
 ```
 Back up to a specific directory in the under storage.
 ```
 ./bin/alluxio fsadmin backup /alluxio/special_backups
-Successfully backed up journal to hdfs://mycluster/opt/alluxio/backups/alluxio-backup-2018-5-29-1527644810.gz
+Successfully backed up journal to hdfs://mycluster/alluxio/special_backups/alluxio-backup-2018-5-29-1527644810.gz
 ```
 Back up to a specific directory on the leading master's local filesystem.
 ```
 ./bin/alluxio fsadmin backup /opt/alluxio/backups/ --local
-Successfully backed up journal to file:///opt/alluxio/backups/alluxio-backup-2018-5-29-1527644810.gz on master Master2
+Successfully backed up journal to /opt/alluxio/backups/alluxio-backup-2018-5-29-1527644810.gz on master Master2
 ```
 
+### checkpoint
+
 ### doctor
 
-The `doctor` command shows Alluxio errors and warnings.
+The `doctor` command gives recommendations and warnings for inconsistent configurations across different alluxio nodes.
```

Comment:
The `doctor` command is used for show Alluxio errors and warnings. Now we not only show inconsistent configuration warnings but also show lost storage warnings.

```
# shows worker lost storage warnings
./bin/alluxio fsadmin storage 
```
lost storage is the storage that worker is not able to access when starting up or running.
Like a worker has a HDD directory in its storage tier and the HDD is unmounted or is deleted. This worker cannot access the HDD directory. A warming will be shown in `fsadmin storage`

Reply:
got it. I'll add this to the docs